experimental/FTheoryTools/src/AbstractFTheoryModels/section_attributes.jl

```diff
@@ -0,0 +1,231 @@
+@doc raw"""
+    defining_classes(m::AbstractFTheoryModel)
+
+Returns the defining divisor classes of the given F-theory model.
```

Comment:
```suggestion
Return the defining divisor classes of the given F-theory model.
```
These haven been unified e.g. by https://github.com/oscar-system/Oscar.jl/pull/3678 to not be in 3rd person singular (also applies to many other places in this PR)

Reply:
Would have been lovely to know. I recently spend a lot of time turning them into "Returns"... But well, thereby this PR gets even larger. Yeah! :+1:

Reply:
Of course appreciate the pointer. :)

Reply:
Likewise, maybe best to attend to this in a separate PR.